db/migrate/20150906173134_create_teams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.belongs_to :account
+      t.belongs_to :team
```

Comment:
from the complete docs it looks like `belongs_to` guesses the association name, while `has_many` guesses the class name. TIL.